l10n_it_fatturapa_in/models/account.py

```diff
@@ -150,8 +150,12 @@ def e_inv_dati_ritenuta(self):
                 "found. Please manually check Withholding tax Amount\n"
             )
         if (
-            sum(self.ftpa_withholding_ids.mapped("amount"))
-            != self.withholding_tax_amount
+            float_compare(
```

Comment:
mi chiedo, se in questo caso, non sia più corretto usare `float_is_zero`, tenendo in considerazione questo [warning](https://github.com/odoo/odoo/blob/689069b0dfe7c61f4e81ef73cbb3436919624610/odoo/tools/float_utils.py#L111-L114).

Reply:
Io leggo il warning al contrario. Usare la `float_compare` e non `float_zero` per confrontare due numeri.

Nell'esempio, `float_compare(0.006, 0.002) == 1` mentre `float_is_zero(0.006 - 0.002) == True`.  0.006 e 0.002 sono diversi (anche in precisione 2, uno vale 0.01 e l'altro 0.00), col test basato su `float_is_zero` li vedrei come uguali (perché la differenza è 0.004 e viene arrotondata a 0.00).